### PR TITLE
nested menus in sidebar

### PR DESCRIPTION
**Pull request by jdittrich**
If pages have a parent, they can now be child items in the sidebar menu. Adjusts template, style and documentation.

To consider: 
* I added information about the nesting to the widget documentation. I was not sure if this is the right place, since the widgets have so far only  config options listed.
* I adjusted the CSS. Please note:The pointer-events:none is gone. I could add it again, but its use is brittle: If inherited, it makes interaction impossible.

### CODE REVIEW

File: exampleSite/content/docs/widgets.md [hunk 1]
```diff
@@ -57,6 +57,14 @@ Here are the configuration options available for the Widgets:
...
     _default: `""`_
 
+The sidebar menu can display nested menus. The sub-pages need to have a page as "parent" defined in the page's front matter:
+
+```yaml
+menu: 
+   main:
+      parent: myParentPage
+```
+
```
Comment (MunifTanjim) [inline exampleSite/content/docs/widgets.md]: Move this to `exampleSite/content/docs/menus.md` under "Sidebar Menu".

And update the `lastmod` front-matter param.

**Comment by MunifTanjim** [comment]
@jdittrich are you comfortable with JavaScript? If you are, then would you like to take a shot adding a toggle button with a `+` or `-` depending on if the submenu is collapsed or expanded?

**Comment by jdittrich** [comment]
> @jdittrich are you comfortable with JavaScript?

Yes – but looking at your code of in [sidebar.js](https://github.com/MunifTanjim/minimo/blob/master/src/scripts/sidebar.js) I don't match your skills in jQuery-free handling the DOM, so I suppose for the sake of code quality and reducing loops,  it makes more sense if you do it.

**Comment by dalliard** [comment]
Have you guys thought about, adding the nested menu also to the normal menu, not only the sidebar menu? Thanks for the great work! 😄

**Comment by MunifTanjim** [comment]
@dalliard Take a look here: https://github.com/MunifTanjim/minimo/issues/13#issuecomment-373878973

@jdittrich It's merged :tada: Thanks for the support.

**Pull request merged**